_includes/releases/v22.2/v22.2.8.md

```diff
@@ -0,0 +1,88 @@
+- Fixed a bug where queries reading from virtual tables (e.g. `crdb_internal.*` and `pg_catalog.*` tables) could hang indefinitely in the rare case when they resulted in an error. [#99968][#99968]
+- In rare cases involving overload and schema changes, users could sometimes, transiently, see errors of the form `"deadline below read timestamp is nonsensical; txn has would have no chance to commit"`. These errors carried an internal pgcode and could not be retried. This form of error is now classified as a retriable error and will be retried automatically either by the client or internally. [#100255][#100255]
+- Fixed a bug where the `sql.mem.distsql.current` metric would double count the memory usage of remote DistSQL flows. [#100254][#100254]
+- Fixed a bug that could prevent a cached query with a [user-defined type](../v22.2/create-type.html) reference from being invalidated, even after a schema change that should prevent the type from being resolved. [#100223][#100223]
+- Fixed a bug that could prevent a cached query from being invalidated when a [user-defined function](../v22.2/user-defined-functions.html) referenced by that query was altered or dropped. [#100223][#100223]
+- Fixed a bug where [user-defined functions](../v22.2/user-defined-functions.html) were introduced that could cause a function call to resolve to the wrong function after changes to the schema search path. [#100223][#100223]
+- Fixed a rare bug existing since before v22.1 that could cause a projected expression to replace column references with the wrong values. [#97593][#97593]
+- Fixed a bug where concurrent DML queries could corrupt an existing primary index when [adding a new column](../v22.2/alter-table.html) with a new column family. If a rollback occurs the table may no longer be accessible. [#100187][#100187]
+
+<h3 id="v22-2-8-performance-improvements">Performance improvements</h3>
+
+- Removed prettify usages that could cause OOM on [Statements](../v22.2/ui-statements-page.html) and [Transactions](../v22.2/ui-transactions-page.html) page. [#99453][#99453]
+- Audit logging could incur extra latency when resolving table/view/sequence names. To address this, this patch adds support for using the lease cache for these looks up and eliminate any extra round trips. [#99661][#99661]
+- The execution of multiple [`FOREIGN KEY`](../v22.2/foreign-key.html) and [`UNIQUE`](../v22.2/unique.html) constraint checks can be parallelized in some cases. As a result, these checks can be completed faster, especially so in multi-region environments where the checks require cross-region reads. The feature is enabled by default on v23.1 release, and in order to enable it on v22.2, one must change the private `sql.distsql.parallelize_checks.enabled` [cluster setting](../v22.2/cluster-settings.html) to `true`. [#100520][#100520]
```

Comment:
```suggestion
- The execution of multiple [`FOREIGN KEY`](../v22.2/foreign-key.html) and [`UNIQUE`](../v22.2/unique.html) constraint checks can be parallelized in some cases to help them complete more quickly. faster, especially so in multi-region environments where the checks require cross-region reads. This feature is disabled by default. To enable it, set the private `sql.distsql.parallelize_checks.enabled` [cluster setting](../v22.2/cluster-settings.html) to `true`. [#100520][#100520]
```
My understanding is that we don't release note anything about multiregion. Can you verify this and also what is a "private" cluster setting?

Reply:
Do you mean "multi-tenant"? We do document MR, but not multi-tenant.

Verifying "private" cluster setting now.